app/models/dqt_no_training_instance_error.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+class DqtNoTrainingInstanceError < StandardError; end
```

Comment:
Not sure there's value in having this as a separate file. Thinking we could put this inside `Dqt::RetrieveTrainingInstance`, so we get `Dqt::NoTrainingInstanceError`.

Reply:
done!